model_analyzer/config/generate/model_run_config_generator.py

```diff
@@ -109,3 +113,11 @@ def _generate_model_run_config(self, model_config, perf_analyzer_config):
                                     perf_analyzer_config)
 
         return run_config
+
+    def _determine_early_exit_enables(self, config, model):
```

Comment:
Please review this function.
Also, I need to update docs once we agree on the exact functionality

Reply:
This doesn't look right to me. Here's what I'm expected

- PACG early exit if enabled and no concurrency list
- MACG early exit if enabled and no config parameters set

In this code we'd enable PACG early exit if enabled, regardless of the presence of a concurrency list (and also enable early exit when disabled and there is no concurrency list).